std/os/darwin.zig

```diff
@@ -676,6 +741,21 @@ pub fn fork() usize {
     return errnoWrap(c.fork());
 }
 
+///fcntl with zero parameters
+pub fn fcntlZero(fildes: i32, cmd: i32) usize {
```

Comment:
Should we use 0, 1, 2 instead of "Zero", "One", "Two" to match the pattern set by `syscall1`, `syscall2`, etc?

Reply:
Done in 19f6dd8